src/django_peertube_runner_connector/models.py

```diff
@@ -0,0 +1,263 @@
+"""Models for the django-peertube-runner-connector app."""
+from datetime import timedelta
+import logging
+from uuid import uuid4
+
+from django.db import models
+from django.utils import timezone
+
+from django_peertube_runner_connector.storage import video_storage
+
+
+logger = logging.getLogger(__name__)
+
+
+class RunnerRegistrationToken(models.Model):
+    """Model representing a PeerTube runner registration token."""
+
+    registrationToken = models.CharField(max_length=255, unique=True)
```

Comment:
I think we should add `help_text` to each attributes, to get a more clear administration view.